cogs/quotes.py

```diff
@@ -22,12 +24,50 @@ async def quote(self, ctx, search=""):
     @quote.command()
     @commands.has_permissions(administrator=True)
     async def add(self, ctx, *, content):
+        """adds a quote"""
+
         content = await commands.clean_content(fix_channel_mentions=True).convert(ctx, content)
 
         with self.bot.db.get(ctx.guild.id) as db:
-            db.execute("INSERT INTO quotes (content) VALUES (?)", (content, ))
+            db.execute("INSERT INTO quotes (content) VALUES (?)", (content,))
 
         await ctx.message.add_reaction('\U00002705')
 
+    @quote.command()
+    @commands.has_permissions(manage_channels=True)
+    async def list(self, ctx, search=""):
+        """Lists all quotes that use an optional query"""
+
+        # if else for performance reasons, as SQLite does not perform query optimisation
+        if len(search) > 0:
+            search = "%" + search + "%"
+
+            with self.bot.db.get(ctx.guild.id) as db:
+                quotes = db.execute("SELECT content FROM quotes WHERE LOWER(content) LIKE ? ORDER BY content",
+                                    (search,)).fetchall()
+        else:
+            with self.bot.db.get(ctx.guild.id) as db:
+                quotes = db.execute("SELECT content FROM quotes ORDER BY content",
+                                    (search,)).fetchall()
+
+        if len(quotes) == 0:
+            await ctx.send("No quotes found.")
+            return
+
+        text = ""
+        for quote in quotes:
+            line = "{}\n".format(quote[0])
+            # lines can never be >= 2000-6, due to being inputted by the user in discord as well using the same
```

Comment:
Comment can be ommitted. Or added as a short `(a single line can't ever be > 1994 chars)` below the other comment.